packages/@react-aria/radio/src/useRadio.ts

```diff
@@ -24,7 +24,15 @@ interface RadioAriaProps extends RadioProps {
 }
 
 interface RadioAria {
-  inputProps: AllHTMLAttributes<HTMLInputElement>
+  inputProps: HTMLAttributes<HTMLElement> & {
```

Comment:
Do I need to break out this inline type definition and make it an interface?

Reply:
You could use `InputHTMLAttributes<HTMLInputElement>` for this one since it's meant for a specific HTML element. That should include all of those extra props you added.